assets/api/examples/event.js

```diff
@@ -0,0 +1,6 @@
+{
+    "eventType": "UPVOTE",
+    "playlistId": 2
```

Comment:
I know this is just an example, but I want to make sure, are we using numeric IDs or UUIDs?